### PR TITLE
fix: добавлен реакт импорт

### PR DESCRIPTION
**Pull request by dontuse**
чиним билд

### CODE REVIEW

File: dist/assets-table.bundle.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /******/ 	}
 /******/ 	
 /******/ 	var hotApplyOnUpdate = true;
-/******/ 	var hotCurrentHash = "2c83443ac09d1a2023e9"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentHash = "164b31558a19e84d7d47"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/assets-table.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Mixed double and single quotes.

**Comment by dontuse** [comment]
@Kupov

File: dist/app.bundle.js [hunk 2]
```diff
@@ -78,7 +78,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /******/ 	}
 /******/ 	
 /******/ 	var hotApplyOnUpdate = true;
-/******/ 	var hotCurrentHash = "2c83443ac09d1a2023e9"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentHash = "c6389545542b96dc3d2c"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/app.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Mixed double and single quotes.

File: dist/assets-table.bundle.js [hunk 3]
```diff
@@ -78,7 +78,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /******/ 	}
 /******/ 	
 /******/ 	var hotApplyOnUpdate = true;
-/******/ 	var hotCurrentHash = "2c83443ac09d1a2023e9"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentHash = "c6389545542b96dc3d2c"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/assets-table.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Mixed double and single quotes.

**Pull request merged**